app/router.js

```diff
@@ -7,7 +7,12 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+
+  this.route('/');
```

Comment:
What's going on with this guy? Are you using the index route for something in particular?